src/spectrecoff/Layout.fs

```diff
@@ -15,25 +15,13 @@ type Padding =
     | HorizontalVertical of int*int
     | TopRightBottomLeft of int*int*int*int
 
-type Style = 
-    | Bold
-    | Dim
-    | Italic
-    | Underline
-    | Invert
-    | Conceal
-    | Slowblink
-    | Rapidblink
-    | StrikeThrough
+open Spectre.Console
+type Look = 
+    { Decoration: Decoration list;
+      Color: Color }
 
-let stringifyStyle style = 
-    match style with
-    | Bold -> "bold"
-    | Dim -> "dim"
-    | Italic -> "italic"
-    | Underline -> "underline"
-    | Invert -> "invert"
-    | Conceal -> "conceal"
-    | Slowblink -> "slowblink"
-    | Rapidblink -> "rapidblink"
-    | StrikeThrough -> "strikethrough"
\ No newline at end of file
+let private aggregate decorations =
+    decorations |>  List.reduce (|||)
+
+let toStyle look = 
+    Style (look.Color, System.Nullable(), aggregate look.Decoration)
```

Comment:
I wonder if we should not call our concept `Style` too, since it will be practically the same as in the `Spectre.Console` world, especially once we implement the background, as you suggested in the description of this PR. What was your reasoning for calling it `Look` instead?

On another note, I think the module name `Layout` is not correct anymore, since the members now not only deal with layout. Again, maybe `Style` is more suitable. After all, `CSS` are cascading *style* sheets, and they deal with layout and looks.

Reply:
This one is tough, I'll try it out, but I wanted to avoid collisions when both ns are open, needing prefixes, etc. Maybe it works well.

Reply:
As discussed in chat, `Style` conflicts too badly when used in conjunction with the Spectre.Console namespace open (needed for, e.g., for `Color `and `Decoration`). So for this PR we'll keep `Look` (until we find a better name, hehe)